chapters/equations.tex

```diff
@@ -398,16 +398,33 @@ \subsubsection{Single Assignment Rule Applied to When-Equations}\label{applicati
 \begin{lstlisting}[language=modelica]
 model WhenPriority
   Boolean close;   // Correct model: close defined by two equations!
-algorithm
+equation
   $\ldots$
   when condition1 then
-    close := true;
+    close = true;
   elsewhen condition2 then
-    close := false;
+    close = false;
   end when;
   $\ldots$
 end WhenPriority;
 \end{lstlisting}
+An alternative to \lstinline!elsewhen! is to use an algorithm.
+However, both statements will be executed if both conditions become \lstinline!true! at the same time.
+Therefore they must be in reverse order to preserve the priority, and any side-effect would require more care.
```

Comment:
@casella Should it be explicitly called out/clarified that an algorithm with an elsewhen can _also_ be used to implement priority (as demonstrated by `WhenPriority` up until now)? This eliminates the need for the reverse-order technique or side-effect considerations.

Reply:
The idea was more to demonstrate the difference between elsewhen and two when-statements, to clarify the semantics. However, it could be clarified that elsewhen can also be used in algorithms. I'll propose something.